.pre-commit-config.yaml

```diff
@@ -82,7 +82,7 @@ repos:
         name: mypy-tests
         args: [--ignore-missing-imports]
         files: ^(packages/|docs|/conftest.py|src/tests)
-        exclude: (^packages/.*/setup.py|/src|^packages/aiohttp/aiohttp_patch.py$)
+        exclude: (^packages/.*/setup.py|/src|^packages/aiohttp/aiohttp_patch.py$|^packages/zfpy/test_zfpy.py$)
```

Comment:
I've been compelled to add this because I adapted most of https://github.com/LLNL/zfp/blob/1.0.1/tests/python/test_numpy.py without much changes and the type annotations there are non-existent. As a result, it's quite messy out here without excluding the file.